src/main/java/task/Deadline.java

```diff
@@ -0,0 +1,22 @@
+package task;
+
+public class Deadline extends Task {
+    private static final String DELIMINATOR = "/ by";
+
+    public Deadline(String description) {
+        super(description);
+    }
+
+    public String getTypeOfTask() {
+        return "D";
+    }
+
+    public String getFormattedDescription() {
+        String[] slicedDescription = description.split(DELIMINATOR);
+        String what = slicedDescription[0];
+        String when = slicedDescription[1].trim();
+        String[] slicedWhen = when.split(" ");
+        when = slicedWhen[0] + ": " + slicedWhen[1];
+        return what + "(" + when + ")";
```

Comment:
Should the variables be named something like "deadlineDescription" and "deadlineTime" instead of "what" and "when"? The naming is quite hard to understand. Thank you.